lte/gateway/c/oai/common/amf_default_values.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/****************************************************************************
+  Source      amf_default_values.h
+  Version     0.1
+  Date        2020/07/28
+  Product     NGAP stack
+  Subsystem   Access and Mobility Management Function
+  Description Defines and Access default values
+
+*****************************************************************************/
+
+#ifndef FILE_AMF_DEFAULT_VALUES_SEEN
+#define FILE_AMF_DEFAULT_VALUES_SEEN
+
+/*******************************************************************************
+ * Timer Constants
+ ******************************************************************************/
+#define AMF_STATISTIC_TIMER_S (60)
+
+/*******************************************************************************
+ * NGAP Constants
+ ******************************************************************************/
+
+#define NGAP_PORT_NUMBER (38412)  ///< NGAP SCTP IANA ASSIGNED Port Number
```

Comment:
statement is not readable. can you make it something like "IANA assigned port number for NGAP payloads on SCTP endpoint"